pipeline_dp/dp_engine.py

```diff
@@ -533,7 +534,67 @@ def _drop_partitions_under_threshold(self, col):
                                     lambda row: row[1].privacy_id_count != None,
                                     "Drop partitions under threshold")
 
-    def _annotate(self, col, params: pipeline_dp.SelectPartitionsParams,
+    def anonymize_values(
```

Comment:
IIRC we refrain from using term "anonymize" or "anonymization" in the tooling, because DP != Anonymization. So that should probably be "add_dp_noise" or apply_dp_to_preaggregated_data?

If you agree, this applies to other placed where you mention anonymization.

Reply:
Good point, I've renamed to `add_dp_noise`